mail-filter/spamassassin/files/4.0.0-tests-dnsbl_subtests.t.patch

```diff
@@ -0,0 +1,13 @@
+--- old/t/dnsbl_subtests.t	2022-12-14 01:03:25.000000000 -0500
```

Comment:
- Please use a/ and b/ instead of old/, new/ and strip the dates (scrub-patch from iwdevtools can do this)
- Please include the respective Bugzilla links at the top of each patch

Thanks!

Reply:
Done.

Learning new stuff everyday, @thesamesam!  Thanks!